arkane/statmech.py

```diff
@@ -357,7 +359,7 @@ def load(self, pdep=False, plot=False):
                                  'in the specified path {1}'.format(self.species.label, statmech_log.path))
             else:
                 statmech_log.path = modified_statmech_path
-        if (isinstance(statmech_log, ESSAdapter) or isinstance(statmech_log, Log)):
+        if isinstance(statmech_log, ESSAdapter):
             statmech_log = ess_factory(statmech_log.path)
```

Comment:
It looks like `statmech_log`, `geom_log`, and `scan_log` all do a modified path check and then call ess_factory. Given this, I think that we can assign things like `Scan_Log` in local context to point towards the new function `create_log`. If we do this we might be able to delete lines like this entirely